portingdb/htmlreport.py

```diff
@@ -138,13 +131,11 @@ def hello():
         breadcrumbs=(
             (url_for('hello'), 'Python 3 Porting Database'),
         ),
-        collections=collections,
         coll_info=coll_info,
```

Comment:
`coll_info` is also not used.
And would be nice to remove `collections` and `coll_info` variables from the function as well.